portal/templates/assignments/teacher-assignments.html

```diff
@@ -0,0 +1,22 @@
+{% extends 'layouts/base.html' %}
+
+{% block title %} Assignments{% endblock %}
+
+
+{% block content %}
+<form method='post'>
+  {% for student in students%}
+  <h2>{{ student['first_name']}}--{{student['last_name']}}</h2>
+  <h3>Assignment name</h3>
+  <h4>{{ student['name']}}</h4>
```

Comment:
Unless you have a good reason we generally shouldn't use h4 or below. If this heading has the same importance as the h3, make it an h3.